Hymns/Display/ShareSheet.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+struct ShareSheet: UIViewControllerRepresentable {
```

Comment:
Could you document what you followed to make this class? Just for posterity

Reply:
Added in